shared/api/index.js

```diff
@@ -14,10 +14,13 @@ const runtimesApi = require("./runtimes");
 // Registry
 const RegistryApi = require("./registry");
 
+const version = "0.4.13";
```

Comment:
For reviwers, I'm not the biggest fan of a hardcoded version but I don't like that the repo will drift from what we upload on npm if we do a search and replace from the CI